src/xmagics/execution.cpp

```diff
@@ -133,15 +144,15 @@ namespace xcpp
             std::vector<double> all_runs;
             double mean = 0;
             double stdev = 0;
-            for (std::size_t r = 0; r < repeat; ++r)
+            for (std::size_t r = 0; r < (std::size_t)repeat; ++r)
```

Comment:
repeat has been defined as a `size_t` variable, no need to cast here.

Reply:
Not anymore.

Reply:
Quick nitpick. We should probably not used "c-style" casting syntax.